contracts/src/periphery/EigenDACertVerifierV1.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierV1} from "src/interfaces/IEigenDACertVerifierV1.sol";
+import {IEigenDAThresholdRegistry} from "../interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDABatchMetadataStorage} from "../interfaces/IEigenDABatchMetadataStorage.sol";
+import {IEigenDASignatureVerifier} from "../interfaces/IEigenDASignatureVerifier.sol";
+import {EigenDACertVerificationV1Lib as CertV1Lib} from "src/libraries/EigenDACertVerificationV1Lib.sol";
+import {EigenDACertVerificationV2Lib as CertV2Lib} from "src/libraries/EigenDACertVerificationV2Lib.sol";
+import {OperatorStateRetriever} from "../../lib/eigenlayer-middleware/src/OperatorStateRetriever.sol";
+import {IRegistryCoordinator} from "../../lib/eigenlayer-middleware/src/RegistryCoordinator.sol";
+import {IEigenDARelayRegistry} from "../interfaces/IEigenDARelayRegistry.sol";
+import "../interfaces/IEigenDAStructs.sol";
+
+/**
+ * @title A CertVerifier is an immutable contract that is used by a consumer to verify EigenDA blob certificates
+ * @notice For V2 verification this contract is deployed with immutable security thresholds and required quorum numbers,
```

Comment:
You mention `V2`, but this is the `V1` verifier. Shouldn't `V2` specific docs live there?

Reply:
It is even worse than you say, as the entire comment is pretty much incorrect and does not apply to V1 verification. I will update this.

Reply:
I'd argue this contract is literally irrelevant now since we have a V1 cert verifier pinned at release 0.1.0 which has production audit guarantees

Reply:
I agree with the irrelevance of this contract, but IMO it is better to avoid removal in this PR to make it easier to see that no important functionality was removed in this PR.

Reply:
@litt3 comment removed